docs/source/en/_toctree.yml

```diff
@@ -33,7 +33,9 @@
     - local: converting_tensorflow_models
       title: Converting from TensorFlow checkpoints
     - local: serialization
-      title: Export 🤗 Transformers models
+      title: ONNX
```

Comment:
Should the ONNX and TorchScript sections be nested under a general "Export 🤗 Transformers models"? Otherwise, the ToC looks a bit weird to me:

![Screen Shot 2022-10-03 at 13 08 33](https://user-images.githubusercontent.com/26859204/193563000-05907664-59e5-42c9-97a6-1b35f02b0de4.png)

Alternatively, we could have "Export to ONNX" and "Export to TorchScrip"

Reply:
Good point, I think `Export to ...` is clearer! 👍